### ISSUE TITLE
Please add support for "SHIZUKU" app for root access

### ISSUE THREAD

**Issue by Manish8580kumar** [NeoApplications/Neo-Backup#785]

**Comment by hg42** [comment]
no more description?

I mean, we know what you want, but what is different to all the discussions we had before?

E.g.

Matrix Bridge, [6/1/23 4:04 PM]
> how does shizuku work
machiav3lli [= Antonios]: It uses a security hole to get some privilaged permissions using adb, as aforementioned, that's even less than being a a privilaged system app and far less than what a backup app needs to function.

Tilo, [6/1/23 4:26 PM]
On some ROMS like LineageOS there is an option for root-debugging so adb allows running commands as root.
That could maybe be an option. But you somehow need to tell NB to use Shizuku and Shizuku to use adb root so that will be brittle

Antonios, [6/1/23 4:52 PM]
tldr; this doesn't really function on a user build without granting root, which is the problem the users trying to avoid → no real way to avoid "root".

Here's two posts pointing to the technical details:
- https://android.stackexchange.com/questions/207623/what-special-privileges-system-xbin-su-does-have-w-r-t-root-access
- https://android.stackexchange.com/questions/213116/why-does-adb-root-do-nothing


first read:
https://github.com/NeoApplications/Neo-Backup/blob/main/FAQ.md#do-i-need-a-rooted-phone
https://github.com/NeoApplications/Neo-Backup/blob/main/FAQ.md#what-is-root-access-used-for


Basically, the suggestion is pointless, unless you provide new information, how to use Shuizuku to access the app data (e.g. `/data/user/0/`_the.package.name_).

**Comment by hg42** [comment]
btw. if you are on this track:

> It is a good feature because it is possible to backup data of some games that do not save data to data/data but to android/data 🙂

(only the apks and some data from internal storage, like `/sdcard/Android`)

then you can copy these by using a file manager (or shell script or similar)
